### PR TITLE
[IMP] Add smaciasosi to Mexico

### PR DESCRIPTION
**Pull request by max3903**
Here is his profile: @SMaciasOSI

### CODE REVIEW

File: conf/psc/field.yml [hunk 1]
```diff
@@ -3,9 +3,9 @@ field-service-maintainers:
     - max3903
     - wolfhall
     - hparfr
-    - smangukiya
     - brian10048
     - osi-scampbell
+    - smacias-osi
```
Comment (max3903) [inline conf/psc/field.yml:8]:
```suggestion
    - smaciasosi
```

File: conf/psc/l10n.yml [hunk 2]
```diff
@@ -349,6 +349,7 @@ local-macedonia-maintainers-psc-representative:
 local-mexico-maintainers:
   members:
     - nhomar
+    - smacias-osi
```
Comment (max3903) [inline conf/psc/l10n.yml:352]:
```suggestion
    - smaciasosi
```

File: conf/psc/management.yml [hunk 3]
```diff
@@ -3,6 +3,7 @@ management-systems-maintainers:
     - max3903
     - ivantodorovich
     - eugen-don
+    - smacias-osi
```
Comment (max3903) [inline conf/psc/management.yml:6]:
```suggestion
    - smaciasosi
```

**Review by etobella — COMMENTED** [review]
@max3903 Seeing his collaborations, I think it is too soon to make him PSC on 3 different areas at the same time. IMO, we need to wait some time before doing this.

Also, the idea is that he should present himself with the motivations to be a PSC

The only one I would maintain is mexico, as we only have one PSC.

**Comment by HaraldPanten** [comment]
I agree with Enric. Maybe we should start with Mexico. Other PSC areas could be assigned in a future.

File: conf/psc/field.yml [hunk 4]
```diff
@@ -3,9 +3,9 @@ field-service-maintainers:
     - max3903
     - wolfhall
     - hparfr
-    - smangukiya
     - brian10048
     - osi-scampbell
+    - smaciasosi
```
Comment (max3903) [inline conf/psc/field.yml:8]:
```suggestion
```

**Comment by simahawk** [comment]
@HaraldPanten @etobella good for you?

**Comment by etobella** [comment]
Sure! Should we squash all the commits on a single one?

**Review by HaraldPanten — APPROVED** [review]
LGTM 👍

**Pull request merged**